src/main/java/thestonedturtle/runiterocks/ui/TableRow.java

```diff
@@ -115,6 +119,18 @@ public void mouseExited(MouseEvent mouseEvent)
 			{
 				setBackground(lastBackground);
 			}
+
+			@Override
+			public void mouseClicked(MouseEvent mouseEvent)
+			{
+				if (plugin.config.leftClickToHop()) {
+					hopToWorld.accept(world);
+				} else if (plugin.config.doubleLeftClickToHop()) {
+					if ((System.currentTimeMillis - lastTimeClicked) <= 500)
```

Comment:
Add the curly braces here and update the rest to match the style of the file.

Since leftClickHop is being removed you could also combine these two if conditions and just always update lastTimeClicked regardless of config option